applications/serial_lte_modem/src/slm_at_tcp_proxy.c

```diff
@@ -228,25 +229,25 @@ static int do_tcp_server_stop(void)
 		proxy.sec_tag = INVALID_SEC_TAG;
 	}
 #endif
+	server_stop_pending = true;
 	if (proxy.sock_peer != INVALID_SOCKET) {
 		(void)close(proxy.sock_peer);
 		proxy.sock_peer = INVALID_SOCKET;
-	}
-	ret = close(proxy.sock);
-	if (ret < 0) {
-		LOG_WRN("close() failed: %d", -errno);
-		ret = -errno;
+		sprintf(rsp_buf, "\r\n#XTCPSVR: 0,\"disconnected\"\r\n");
```

Comment:
It'll be printed in case of POLLNVLD, for example, and we want to keep it the same, i.e.
#XTCPSVR: 0,"disconnected"
#XTCPSVR: 0,"stopped"
for both intentional and interrupted TCP server shutdown